python/lancedb/table.py

```diff
@@ -776,6 +780,7 @@ def create(
         if data is not None:
             table.add(data)
 
+        EVENTS("create_table")
```

Comment:
Potential places to instrument:

1. LanceDB.Connect
2. Drop_table
3. Delete
4. Update
5. Merge
6. Restore